assembly.py

```diff
@@ -338,6 +339,27 @@ def parser_assemble_trinity(parser=argparse.ArgumentParser()):
 
 __commands__.append(('assemble_trinity', parser_assemble_trinity))
 
+def gapfill_gap2seq(in_scaffold, inBam, out_scaffold, gap2seq_opts='', threads=1, mem_limit_gb=4):
+    ''' This step runs the Gap2Seq tool to close gaps between contigs in a scaffold.
+    '''
+    tools.gap2seq.Gap2SeqTool().gapfill(in_scaffold, inBam, out_scaffold, gap2seq_opts=gap2seq_opts, threads=threads,
+                                        mem_limit_gb=mem_limit_gb)
+
+def parser_gapfill_gap2seq(parser=argparse.ArgumentParser(description='Close gaps between contigs in a scaffold')):
+    parser.add_argument('in_scaffold', help='Scaffold with gaps (FASTA witht Ns)')
+    parser.add_argument('inBam', help='Input unaligned reads, BAM format.')
+    parser.add_argument('out_scaffold', help='Output assembly.')
+    parser.add_argument('--gap2seq_opts', default='', help='(advanced) Extra command-line options to pass to Gap2Seq')
```

Comment:
We have a loose convention of having the argument names exposed via CLI in `--camelCase` with the argparse `dest='snake_case'` (A bit less for the user to type while still striving for PEP 8 within our Python code.)